src/site/content/en/animations/animations-examples/index.md

```diff
@@ -0,0 +1,346 @@
+inspect any of the moving elements in Chrome DevTools.
+For example select the triangle,
+and watch how the box of the element transforms during its journey into the air,
+as it spins, and then returns to the start position.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/triangle.mp4" type="video/webm">
+  </video>
+  <figcaption class="w-figcaption">
+    Video showing how we can track the path of the triangle in Chrome DevTools.
+  </figcaption>
+</figure>
+
+With the element still selected look at the Styles Panel.
+There you can see the CSS which draws the shape of the triangle,
+and the animation being used.
+
+The triangle is created by using the `::after` pseudo-element to add generated content,
+using borders to create the shape.
+
+```css
+.triangle {
+    position: absolute;
+    bottom: -62px;
+    left: -10px;
+    width: 110px;
+    height: 110px;
+    border-radius: 50%;
+}
+
+.triangle::after {
+    content: "";
+    position: absolute;
+    top: 0;
+    right: -10px;
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 0 28px 48px 28px;
+    border-color: transparent transparent #89beb3 transparent;
+}
+```
+
+{% Glitch 'animation-breakdown1-1' %}
+
+{% Aside %}
+You can find out more about making shapes with borders and generated content in
+[The Shapes of CSS](https://css-tricks.com/the-shapes-of-css/).
+{% endAside %}
+
+The animation is added with the following line of CSS,
+
+```css
+animation: path_triangle 10s ease-in-out infinite;
+```
+
+Find the keyframes by scrolling down the Style Panel.
+There you will find that the animation is created by using `transform` to change the position of the element and rotate it.
+The `transform` property is one of the properties described in the [Animations Guide](/animations-guide),
+which does not cause the browser to do layout or paint operations.
+
+```css
+@keyframes path_triangle {
+  0% {
+    transform: translateY(0);
+  }
+  10% {
+    transform: translateY(-172px) translatex(10px) rotate(-10deg);
+  }
+  55% {
+    transform: translateY(-172px) translatex(10px) rotate(-365deg);
+  }
+  58% {
+    transform: translateY(-172px) translatex(10px) rotate(-365deg);
+  }
+  63% {
+    transform: rotate(-360deg);
+  }
+}
+```
+
+{% Glitch 'animation-breakdown1-2' %}
+
+Each of the different moving parts of this animation uses similar techniques.
+The result is a complex animation which runs smoothly.
+
+## Pulsating Circle
+
+{% IFrame {
+  src: 'https://codepen.io/peeke/embed/BjxXZa?height=458&theme-id=light&default-tab=result',
+  title: 'Pulsating Circle by Peeke'
+}
+%}
+
+This type of animation is sometimes used to draw attention to something on a page.
+To understand the animation you can use Firefox DevTools.
+
+With the animation running, open the Performance tab in Firefox DevTools and record a few seconds of the animation.
```

Comment:
```suggestion
To understand the animation you can use Firefox DevTools.

### Inspect the animation with Firefox DevTools

With the animation running, open the Performance tab in Firefox DevTools and record a few seconds of the animation.
```

I wonder if we should mention that we're just showing Chrome & Firefox to demonstrate they can both be used to inspect animations. In some cases a tool might only be available in Chrome or Firefox. But for this page it seems like they have feature parity.